code/qcommon/common.cpp

```diff
@@ -1196,6 +1196,7 @@ void Com_Init( char *commandLine ) {
 				Cbuf_AddText ("cinematic openinglogos\n");
 			}
 		}
+		CL_StartHunkUsers();
```

Comment:
I'm unsure this call is necessary. I only added it so SP and MP match 1:1.

Reply:
That is the only part of what makes the fix happen, the moving of the other two functions has no effect on the sound code.